### PR TITLE
Admin edit update delete show subject

### PR DESCRIPTION
**Pull request by TuyenMary**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -75,3 +75,10 @@ input{
   padding-bottom: 20px;
 }
 
+.subject {
+border: 1px solid #CCCCCC;
```
Comment (DatDaiGia) [inline app/assets/stylesheets/application.css]: ident

File: app/controllers/admin/subjects_controller.rb [hunk 2]
```diff
@@ -22,8 +27,37 @@ def create
...
+    end
+  end
+
   private
   def subject_params
     params.require(:subject).permit :name, :description
   end
+
+  def load_subject
+     @subject = Subject.find params[:id]
```
Comment (DatDaiGia) [inline app/controllers/admin/subjects_controller.rb]: ident

File: app/views/admin/subjects/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<% provide :title, @subject.name %>
+  <div class="devise-content">
+  <div class="row">
+    <aside class="col-md-4">
+      <section>
+        <div class="subject">
+          <p id="name"><%=  @subject.name %></p>
+          <p><%=  @subject.description %><p>
```
Comment (DatDaiGia) [inline app/views/admin/subjects/show.html.erb]: 1 space

File: app/views/admin/subjects/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<% provide :title, @subject.name %>
+  <div class="devise-content">
+  <div class="row">
+    <aside class="col-md-4">
+      <section>
+        <div class="subject">
+          <p id="name"><%=  @subject.name %></p>
```
Comment (DatDaiGia) [inline app/views/admin/subjects/show.html.erb]: 1 space

**Comment by DatDaiGia** [comment]
ok

**Comment by nhunght** [comment]
OK

**Pull request merged**